### PR TITLE
chore: add info about translations table to getting started

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/getting-started.mdx [hunk 1]
```diff
@@ -432,6 +432,8 @@ You can customize the translations in a form by using the `translations` propert
 
 Alternatively, you can use the global Eufemia [Provider](/uilib/usage/customisation/localization/) (example further down).
 
+You find all the available strings/keys in the properties tab of each field component. You can have a look at the [PhoneNumber field](!/uilib/extensions/forms/feature-fields/PhoneNumber/properties/#translations).
```
Comment (joakbjerk) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/getting-started.mdx:435]:
```suggestion
You can find all available strings/keys in the properties tab of each field component. You can have a look at the [PhoneNumber field](!/uilib/extensions/forms/feature-fields/PhoneNumber/properties/#translations).
```

**Review by joakbjerk — APPROVED** [review]
⭐️

**Review by langz — APPROVED** [review]
👌

**Pull request merged**